cuml/svm_demo.ipynb

```diff
@@ -24,15 +26,20 @@
     "import cuml.svm \n",
     "import sklearn.svm\n",
     "\n",
-    "from sklearn.datasets.samples_generator import make_gaussian_quantiles\n",
-    "from sklearn.model_selection import train_test_split"
+    "from sklearn.datasets import make_gaussian_quantiles\n",
+    "from sklearn.datasets import make_friedman1\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from cuml.metrics.regression import r2_score"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Generate data"
+    "## 2. Support vector classification\n",
+    "Currently cuML supports binary classification (C-Support Vector Classification).\n",
+    "\n",
+    "### Generate data"
```

Comment:
why does the generate data have commented out code?

Reply:
I guess the original idea was to allow switching to linearly separable data just by uncommenting that code. It feels unnecessary now, I have deleted the commented code.